extras/logo/svg/openslides-icon.svg

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 15.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
```

Comment:
I would say: same like OpenSlides core (MIT).
I could reference to this url: http://opensource.org/licenses/MIT
Ok?